src/style.css

```diff
@@ -1 +1,132 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
```

Comment:
The font-weight for 'Avenir-Heavy' is set to 400, which is usually 'normal'. If 'Avenir-Heavy' is intended to be a bold font, consider using a higher weight, such as 700.